tests/e2e/tests/web/regression/insights/feature-flag.e2e.ts

```diff
@@ -24,7 +24,7 @@ const pathes = {
     flagOffConfig: path.join('.', 'test-data', 'features-configs', 'insights-flag-off.json')
 };
 
-fixture `Feature flag`
+fixture.skip `Feature flag`
```

Comment:
let's add here the reason why we skip this test as comment
like link to the related story
to not forget for the future

Reply:
added